cypher_most_mentioned_people.go

```diff
@@ -21,24 +20,21 @@ func (pcw CypherDriver) MostMentioned(fromDateEpoch int64, toDateEpoch int64, li
 		Parameters: neoism.Props{"fromDateEpoch": fromDateEpoch, "toDateEpoch": toDateEpoch, "mentionsLimit": limit},
 		Result:     &results,
 	}
-	log.Infof("Query %v", query)
+	log.Debugf("Query %v", query)
 
-	err = pcw.db.Cypher(query)
-	log.Infof("Results%v", &results)
-
-	if err != nil {
+	if err = pcw.conn.CypherBatch([]*neoism.CypherQuery{query}); err != nil {
 		log.Errorf("Error finding %v most mentioned people between %v and %v with the following statement: %v  Error: %v", limit, fromDateEpoch, toDateEpoch, query.Statement, err)
-		return []Thing{}, false, fmt.Errorf("Error finding %v most mentioned people between %v and %v", limit, fromDateEpoch, toDateEpoch)
+		return []Thing{}, false, err
```

Comment:
Same as somewhere above :smile: , shouldn't the case when found is false, and no error, be covered somewhere in this method?